src/appLoader.js

```diff
@@ -13,7 +13,6 @@ import Main from 'src/pages/Main';
 const rootElement = document.getElementById('root');
 
 RModal.defaultStyles = { overlay: {}, content: {} };
-RModal.setAppElement(rootElement);
```

Comment:
React Modal can add "aria-hidden" to the "app element" when a modal is opened. We don't use that feature (React Modal's `ariaHideApp ` props is always set to false), so we don't need to configure this.